schema/user.js

```diff
@@ -0,0 +1,16 @@
+const userSchema = `
```

Comment:
should events also have a resolver + schema?

Reply:
yep! just did the user one first to see how it works. I can do the events one tonight